proposals/simd/SIMD.md

```diff
@@ -674,6 +679,12 @@ Load a `v128` vector from the given heap address.
 
 Load a single element and splat to all lanes of a `v128` vector.
 
+```python
+def S.load_splat(memarg):
+    val = memory[memarg.start:memarg.start + S.LaneBytes])
+    return S.splat(S.LaneType.from_bytes(val))
```

Comment:
I don't think we need `S.LaneType.from_bytes(val)`, val is already a value of the right type (since we load S.LaneBytes of it). And the `S` and `S.splat` has the right lane type associated.

Reply:
I assume that `memory` is a byte array (maybe it is better to call it `linearMemory` or something like that), that's why `val` is a slice of a byte array.  `splat` operation defined in the document would not work on it. I guess it would not hurt to make this a bit more transparent.